app/components/notice/notice.html.erb

```diff
@@ -0,0 +1,3 @@
+<% if notice %>
+  <p id="notice"><%= notice %></p>
```

Comment:
I usually don’t use IDs for styling purposes at all as they usually only introduce specificity issues.

```suggestion
  <div class="Notice"><%= notice %></div>
```